list

```diff
@@ -74,6 +75,7 @@
 9nl.com
 9qr.de
 9splay.store
+9tl.ru
```

Comment:
Looks like I'm having problem to verify this one 🤔

Reply:
Looks like old shortened links I've found all return 404, any new links like the one above return 200 and a page saying "The link you are trying to access has expired either because the campaign has ended or because the link has been disabled."